apps/happy-blocks/src/support-page/inline-skeleton.tsx

```diff
@@ -0,0 +1,26 @@
+import classnames from 'classnames';
+import React from 'react';
+
+type InlineSkeletonProps = {
+	loaded: boolean;
+	hidden?: boolean;
+	large?: boolean;
+	children: React.ReactNode;
+};
+
+/**
+ * Loading skeleton that takes its sizes from the font size
+ */
+export const InlineSkeleton = ( props: InlineSkeletonProps ) => {
```

Comment:
Just curious, don't we already have some kind of inline loading skeleton component within Calypso? Or perhaps it's much more complex and not worth reusing it?

Reply:
We do (and I found your name related to it :) ), but the issue with it is that it is block-based, so need to specify the height for it. And I wanted a skeleton component. that can re-use container line-height.